_layouts/archive.html

```diff
@@ -1,78 +1,84 @@
+        >
+          {{ page.title }}
+        </span>
+        {% endif %}
+        <p class="mt-4 text-justify">
+          {{ site.data.descriptions[page.autopages.display_name] }}
+        </p>
+        <button
+          class="flex items-center w-full sm:w-1/2 xl:w-1/3 mt-12 text-left space-x-3 px-4 h-12 bg-white hover:bg-slate-200 shadow-sm rounded-lg text-gray-500"
+          onclick="showSearch()"
+        >
+          <svg
+            width="24"
+            height="24"
+            fill="none"
+            stroke="currentColor"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          >
+            <path d="m19 19-3.5-3.5"></path>
+            <circle cx="11" cy="11" r="6"></circle>
+          </svg>
+          <span class="flex-auto">Search for posts...</span>
+        </button>
+      </div>
     </header>
 
     <main>
-        <section class="p-6 sm:p-10">
-            <div class="flex flex-col sm:grid sm:grid-cols-2 lg:grid-cols-3 sm:gap-4 md:gap-4 lg:gap-6 text-center">
-                {% for post in paginator.posts %}
-                {% include postbox.html %}
-                {% endfor %}
-            </div>
-            {% if paginator.total_pages > 1 %}
-            <div class="flex justify-center items-center w-full">
-                {% if paginator.previous_page %}
-                <a href="{{ paginator.previous_page_path }}"
-                    class="mr-2 px-4 py-0.5 hover:shadow-xl hover:shadow-slate-200 rounded-md bg-background text-white text-lg hover:scale-110 transition underline-none">
-                    Previous
-                </a>
-                {% endif %}
-                <span>Page {{ paginator.page }} of {{ paginator.total_pages }}</span>
-                {% if paginator.next_page %}
-                <a href="{{ paginator.next_page_path }}"
-                    class="ml-2 px-4 py-0.5 hover:shadow-xl hover:shadow-slate-200 rounded-md bg-background text-white text-lg hover:scale-110 transition underline-none">
-                    Next
-                </a>
-                {% endif %}
-            </div>
-            {% endif %}
-        </section>
-
-        <div class="mb-8">
-            {% include cta.html %}
+      <section class="p-6 sm:p-10">
+        <div
+          class="flex flex-col sm:grid sm:grid-cols-2 lg:grid-cols-3 sm:gap-4 md:gap-4 lg:gap-6 text-center"
+        >
+          {% for post in paginator.posts %} {% include postbox.html %} {% endfor
+          %}
+        </div>
+        {% if paginator.total_pages > 1 %}
+        <div class="flex justify-center items-center w-full">
+          {% if paginator.previous_page %}
+          <a
+            href="{{ paginator.previous_page_path }}"
+            class="mr-2 px-4 ml-2 py-0.5 font-bold rounded-md text-black bg-white hover:bg-slate-300 hover:scale-110 transition underline-none"
```

Comment:
You changed the light one too :)
Light one is correct, just need to fix `dark:` values.

<img width="193" alt="image" src="https://user-images.githubusercontent.com/46792249/162606225-ded00326-f7be-4845-b5f8-e32aeaac9b6d.png">